SimPEG/EM/Analytics/NSEM.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+from scipy.constants import epsilon_0
+from scipy.constants import mu_0
+from SimPEG.EM.Utils.EMUtils import k
+from SimPEG.EM.Utils.EMUtils import omega
+
+__all__ = [
+    'MT_LayeredEarth'
+]
+
+
+# Evaluate Impedance Z of a layer
+_ImpZ = lambda f, mu, k: omega(f)*mu/k
+
+# Complex Cole-Cole Conductivity - EM utils
+_PCC = lambda siginf, m, t, c, f: siginf*(1.-(m/(1.+(1j*omega(f)*t)**c)))
+
+# matrix P relating Up and Down components with E and H fields
+_P = lambda z: np.matrix([[1., 1, ], [-1./z, 1./z]], dtype='complex_')
+_Pinv = lambda z: np.matrix([[1., -z], [1., z]], dtype='complex_')/2.
```

Comment:
Shorthand for 'complex128' (64 bit real and imaginary)
https://docs.scipy.org/doc/numpy-1.12.0/user/basics.types.html